django/contrib/gis/geos/prototypes/geom.py

```diff
@@ -45,6 +45,7 @@ class StringFromGeom(GEOSFuncFactory):
 
 # The GEOS geometry type, typeid, num_coordinates and number of geometries
 geos_normalize = IntFromGeom('GEOSNormalize')
+geos_makevalid = GeomOutput('GEOSMakeValid', argtypes=[GEOM_PTR])
```

Comment:
This block looks ordered alphabetically (although that may be a fluke when compared to the blocks below):
```suggestion
geos_makevalid = GeomOutput('GEOSMakeValid', argtypes=[GEOM_PTR])
geos_normalize = IntFromGeom('GEOSNormalize')
```